addons/mrp/models/mrp_production.py

```diff
@@ -1766,3 +1778,142 @@ def _check_immediate(self):
                     ) and float_is_zero(production.qty_producing, precision_digits=pd):
                 immediate_productions |= production
         return immediate_productions
+
+    def action_serial_mass_produce_wizard(self):
+        self.ensure_one()
+        self._check_company()
+        if self.state != 'confirmed':
+            return
+        if self.product_id.tracking != 'serial':
+            return
+        have_serial_components, missing_components, multiple_lot_components = self._check_serial_mass_produce_components()
+        if have_serial_components:
+            raise UserError(_("Can't work with components tracked by serial number"))
```

Comment:
Is it possible to have this error from the UI? If not it's probably a better idea to raise a not implemented error for serial numbers

Reply:
removed, remaining test needed when entry was by action menu